src/utilities/util.ts

```diff
@@ -435,25 +435,23 @@ export async function generateProjectDropDown(nslist?: Array<string>): Promise<s
  * @returns - A Promise containing the list of parameters to pass to the command
  */
 export async function requestLogInInfo(): Promise<string[] | undefined> {
-  let args: Array<string> = [];
+  const validRegex: { [key: string]: RegExp } = {
+    "OC Command": /^(oc login)/,
+    "Auth Token": /([\s]+--token=sha256~[A-Za-z0-9-_]+)/,
+    "Server URL": /([\s]+--server=[A-Za-z0-9-\\\/\._~:\?\#\[\]@!\$&'\(\)\*\+,:;%=]+)/,
+  };
```

Comment:
We should also add support for the `--certificate-authority` flag

![image](https://github.com/IBM/operator-collection-sdk-vscode-extension/assets/24191850/67640588-3b05-4d6e-8349-3e1370db082d)

FYI....I do not plan to support username and password at this time